### PR TITLE
added django-parler model translation package

### PR DESCRIPTION
**Pull request by kcemenike**
Checklist
- [✔] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [✔] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [✔] Source is from official source
- [✔] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [✔] If static libraries are linked in, the license of the static library is packaged.
- [✔ ] Build number is 0
- [✔] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [✔] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/django-parler/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  sha256: f633522c06727852978cf2f92c94a98d900dff47235412e64ed46d63e68b1f09
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```
Comment (scopatz) [inline recipes/django-parler/meta.yaml:19]: Does this need a c-compiler?

File: recipes/django-parler/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        - No ORM query hacks.
+        - Easy to combine with custom Manager or QuerySet classes.
+        - Easy to construct the translations model manually when needed.
+  
+  doc_url: https://django-parler.readthedocs.io/
+  dev_url: https://github.com/mbi/django-parler/
+
+extra:
+  recipe-maintainers:
+    - django-parler
```
Comment (scopatz) [inline recipes/django-parler/meta.yaml:56]: Is this a real user or team?

Comment (kcemenike) [reply recipes/django-parler/meta.yaml:56]: It's a team. You may remove it if teams aren't allowed to be maintainers. Or should I go ahead and make the modification myself?

Thanks @scopatz

File: recipes/django-parler/meta.yaml (hunk 1 above)
Comment (kcemenike) [reply recipes/django-parler/meta.yaml:19]: I apologise, I should remove that.

File: recipes/django-parler/meta.yaml (hunk 2 above)
Comment (scopatz) [reply recipes/django-parler/meta.yaml:56]: Teams can be maintainers, but I think this will resolve to the conda-forge team in its current form

Comment (kcemenike) [reply recipes/django-parler/meta.yaml:56]: alright, I'll be sole maintainer

File: recipes/django-parler/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```
Comment (scopatz) [inline recipes/django-parler/meta.yaml:22]: Is django not a runtime dep?

Comment (kcemenike) [reply recipes/django-parler/meta.yaml:22]: yes it has been added

File: recipes/django-parler/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    - django >=1.7
+
+test:
+  requires:
+    - six
+    - django >=1.7
```
Comment (dopplershift) [inline recipes/django-parler/meta.yaml:28]: You don't need to duplicate the django dependency here--having it above in `run:` will take care of it. Why do you need `six`?

Comment (kcemenike) [reply recipes/django-parler/meta.yaml]: six is usually needed for django 1.7 but less than 2.2 (so it's more like an unspoken dependency).

Both removed as requested.

File: recipes/django-parler/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - django >=1.7
```
Comment (dopplershift) [inline recipes/django-parler/meta.yaml:23]: `six` is indeed a required dependency, as listed in [django-parler's `setup.py`](https://github.com/django-parler/django-parler/blob/126b2a5bb96c44bdddbf7a620477b19e44bfa8ca/setup.py#L41) so you should have it here under `run`.

Comment (kcemenike) [reply recipes/django-parler/meta.yaml:23]: True this, resolved now.

**Pull request merged**